### DISCUSSION TITLE
rust as language

### DISCUSSION THREAD

**Discussion by ceeveetje** [dodona-edu/feedback-and-support#29] [Feature requests]
Is Rust being considered to add to dodona?

Kind rgds,
Chris

**Comment by bmesuere** [comment]
At present, we don't use Rust in our courses, and there are no immediate plans to develop a custom Rust judge. However, Dodona is equipped with a universal judge system capable of supporting new languages, including Rust.

If you would be interested in adding Rust, this guide would be a good starting point: [Add a New Language to Dodona](https://docs.dodona.be/en/references/tested/new-programming-language/). We're here to help if you need further assistance.
